src/test/regress/expected/distributed_locks.out

```diff
@@ -0,0 +1,76 @@
+CREATE SCHEMA distribute_lock_tests;
+SET search_path TO distribute_lock_tests;
+SET citus.next_shard_id TO 10000;
+CREATE TABLE dist_table(a int);
+SELECT create_distributed_table('dist_table', 'a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+INSERT INTO dist_table SELECT n FROM generate_series(1, 5) n;
+-- Test acquiring lock outside transaction
+LOCK dist_table IN ACCESS EXCLUSIVE MODE;
+ERROR:  LOCK TABLE can only be used in transaction blocks
+-- Test acquiring lock inside procedure
+DO $$
+BEGIN
+LOCK dist_table IN ACCESS EXCLUSIVE MODE;
+END;
+$$;
+-- Test that when the user does not have the required permissions to lock
+-- the locks are not forwarded to the workers
+CREATE USER read_only_user;
```

Comment:
Minor: These commands work on enterprise and hence avoids the NOTICE/HINT.

So, we typically do the following to prevent divergence of test outputs on community vs enterprise:
```SQL
SET client_min_messages TO ERROR;
 CREATE USER read_only_user;
RESET client_min_messages ;
```